hamp_radar/serde_flight.py

```diff
@@ -0,0 +1,160 @@
+from collections.abc import Iterable
+from typing import List
+from pathlib import Path
+
+from geometries import (
+    MultiMainBlockGeometry,
+    PdsFileGeometry,
+    DatasetBlockGeometry,
+    DatasetGeometry,
+    FlightGeometry,
+)
+from serde_pds_files import (
+    get_pdsfile_geometries,
+    write_serialized_geometry,
+)
+
+
+def ppar_for_dataset(
+    filename: Path, mmbg: MultiMainBlockGeometry
+) -> DatasetBlockGeometry:
+    """
+    Creates a DatasetBlockGeometry object with only a single PPAR from a MultiMainBlockGeometry
+    instance. Assumes mmbg.tag == "PPAR".
```

Comment:
Actually, the `mmbg.tag` likely isn't "PPAR", because the `mmbg.tag` would be the tag identifying the radar.
Apart from that, I suggest to always assert conditions with a function assumes as a precondition (except if it is way too expensive to do the assert). So "Assumes mmbg.tag == "PPAR"." should result in:
```python
assert mmbg.tag == "PPAR"
```

Reply:
yes good point, I also changed the other error to an assert since it falls under the same category of checking the mmbg is a valid ppar mmbg, is this okay?